__tests__/app.test.js

```diff
@@ -18,24 +18,60 @@ describe("App", () => {
       .get("/api/nonsense")
       .expect(404)
       .then((res) => {
-        console.log(res.body);
         expect(res.body.msg).toBe("Sorry can't find that!");
       });
   });
-});
 
-describe("/api/categories", () => {
-  test("GET 200- responds with an array of category objects", () => {
-    return request(app)
-      .get("/api/categories")
-      .expect(200)
-      .then((res) => {
-        expect(Array.isArray(res.body.categories)).toBe(true);
-        expect(res.body.categories.length).toBe(4);
-        res.body.categories.forEach((category) => {
-          expect(typeof category.slug).toBe("string");
-          expect(typeof category.description).toBe("string");
+  describe("/api/categories", () => {
+    test("GET 200- responds with an array of category objects", () => {
+      return request(app)
+        .get("/api/categories")
+        .expect(200)
+        .then((res) => {
+          expect(Array.isArray(res.body.categories)).toBe(true);
+          expect(res.body.categories.length).toBe(4);
+          res.body.categories.forEach((category) => {
+            expect(typeof category.slug).toBe("string");
+            expect(typeof category.description).toBe("string");
+          });
         });
-      });
+    });
+  });
+
+  describe("/api/reviews/:review_id", () => {
+    test("GET 200- should respond with a review object, of the correct id ", () => {
+      return request(app)
+        .get("/api/reviews/1")
+        .expect(200)
+        .then((res) => {
+          expect(typeof res.body.review[0]).toBe("object");
+
+          expect(typeof res.body.review[0].review_id).toBe("number");
+          expect(typeof res.body.review[0].title).toBe("string");
+          expect(typeof res.body.review[0].category).toBe("string");
+          expect(typeof res.body.review[0].designer).toBe("string");
+          expect(typeof res.body.review[0].owner).toBe("string");
+          expect(typeof res.body.review[0].review_body).toBe("string");
+          expect(typeof res.body.review[0].review_img_url).toBe("string");
+          expect(typeof res.body.review[0].created_at).toBe("string");
+          expect(typeof res.body.review[0].votes).toBe("number");
+        });
+    });
+    test("GET 404 when passed anything other than a number recieve invalid id number", () => {
+      return request(app)
+        .get("/api/reviews/cuppatea")
+        .expect(404)
+        .then((res) => {
+          expect(res.body.msg).toBe("ID must be a number");
+        });
+    });
+    test("GET 404 when nothing is returned recieve not found", () => {
+      return request(app)
+        .get("/api/reviews/2000")
+        .expect(404)
+        .then((res) => {
+          expect(res.body.msg).toBe("Invalid input!");
```

Comment:
Is this an invalid input? Your status code is correct, if you're going to use/test custom error messages they should be more explicit. Here, the review doesn't exist. It isn't an invalid input as the review id is a number.